### PR TITLE
KTL-392 Kotlinlang - Consistent UI - update footer styles

### PR DESCRIPTION
**Pull request by nikpachoo**
- [YT](https://youtrack.jetbrains.com/issue/KTL-392)
- [UI](https://branch-ktl-392-update-footer.kotlin-web-site.labs.jb.gg/)

### CODE REVIEW

File: scripts/react-renderer/compile.mjs [hunk 1]
```diff
@@ -3,8 +3,27 @@ import { renderToString } from 'react-dom/server';
...
-    console.log(renderToString(createElement(Component, props)));
+    let res = '';
+
+    try {
+        res = renderToString(createElement(Component, props))
+    } catch (e) {
+        res = e;
+    }
+
+    console.log(res);
```
Comment (krutilov) [inline scripts/react-renderer/compile.mjs:28]: Is it needed?

**Review by krutilov — APPROVED** [review]

File: scripts/react-renderer/compile.mjs (hunk 1 above)
Comment (zoobestik) [reply scripts/react-renderer/compile.mjs:28]: Actually, yes. [Python takes](https://github.com/JetBrains/kotlin-web-site/blob/master/src/ktl_components.py#L60) node SSR from stdout.

File: static/js/ktl-component/footer/index.jsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import GlobalFooter from '@jetbrains/kotlin-web-site-ui/dist/footer.js';
+import '@jetbrains/kotlin-web-site-ui/dist/footer.css';
```
Comment (zoobestik) [inline static/js/ktl-component/footer/index.jsx:3]: Is not possible case when footer/header not will have styles when js is disabled?

**Review by zoobestik — CHANGES_REQUESTED** [review]

File: static/js/ktl-component/footer/index.jsx (hunk 2 above)
Comment (nikpachoo) [reply static/js/ktl-component/footer/index.jsx:3]: There is nothing to worry about. We use the "ExtractCssPlugin" that extracts all styles into a separate css file ("global.css").

**Review by zoobestik — APPROVED** [review]

**Pull request merged**